src/domains/dashboard/components/node-view/node-view.tsx

```diff
@@ -81,10 +82,10 @@ const SubSection = memo(({
                   {
                     ...attributes,
                     forceTimeWindow: true, // respect timeWindow
+                    aggrMethod: mapDefaultAggrMethod(chartsMetadata.charts[attributes.id].units),
```

Comment:
Nice! But how would we be able to pass in attributes from the cloud now?
That may be because I haven't followed the commonAttributes development. 
So what is their difference with the attributesOverrides?

Reply:
Thanks for asking! It's related to what we discussed [here](https://github.com/netdata/dashboard/pull/142#discussion_r503310322)? That is - `commonAttributesOverrides` , with attributes for specific chart ids should now be pointed only to dygraph charts - that's because gauge and dygraph charts share the same chartIds and there's conflict (like the one with aggrMethod). Do you agree with that? The selected dimensions will still be passed to dygraphs, [here](https://github.com/netdata/dashboard/blob/7d6e115f5204a50d3ba5cc6152cc06a24c96521e/src/domains/dashboard/components/node-view/render-submenu-name.tsx#L121-L155)

Reply:
Yeap I see! Nice work @jacekkolasa ! Thanks for the clarifications.